packages/runtime/blob-cjs-build.mjs

```diff
@@ -0,0 +1,20 @@
+import * as esbuild from '@netlify/esbuild'
+
+// https://github.com/evanw/esbuild/issues/1760#issuecomment-964900401
+const stripNodeColonPlugin = {
+  name: 'strip-node-colon',
+  setup({ onResolve, onLoad }) {
+    onResolve({ filter: /^node:/ }, (args) => {
+      return { path: args.path.slice('node:'.length), external: true }
+    })
+  },
+}
+
+await esbuild.build({
+  entryPoints: ['src/templates/netliblob.mts'],
+  format: 'cjs',
+  bundle: true,
+  outfile: 'src/templates/blob.js',
+  platform: 'node',
+  plugins: [stripNodeColonPlugin],
+})
```

Comment:
this compiles ESM-only `@netlify/blobs` to CJS and removes `node:` prefixes in import/requires (like from `node:fs`) to make resulting code compatible with our minimal node version